docs/UserGuide.md

```diff
@@ -110,15 +115,16 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in FriendDex.
+Shows a list of all persons in FriendDex. 
+Can optionally provide group name to only list all the friends within a group.
```

Comment:
Would this be better? `Can optionally provide a group name to list all friends in that group`